samples/communication-between-nodes/src/main/java/com/badoo/ribs/samples/comms_nodes/rib/language_selector/builder/LanguageSelectorBuilder.kt

```diff
@@ -0,0 +1,39 @@
+package com.badoo.ribs.samples.comms_nodes.rib.language_selector.builder
+
+import com.badoo.ribs.builder.SimpleBuilder
+import com.badoo.ribs.core.modality.BuildParams
+import com.badoo.ribs.samples.comms_nodes.app.Language
+import com.badoo.ribs.samples.comms_nodes.rib.language_selector.LanguageSelector
+import com.badoo.ribs.samples.comms_nodes.rib.language_selector.LanguageSelectorNode
+import com.badoo.ribs.samples.comms_nodes.rib.language_selector.LanguageSelectorPresenter
+import com.badoo.ribs.samples.comms_nodes.rib.language_selector.LanguageSelectorPresenterImpl
+import com.badoo.ribs.samples.comms_nodes.rib.language_selector.LanguageSelectorView
+
+class LanguageSelectorBuilder(
+    private val languages: List<Language>,
+    private val currentLanguage: Language
```

Comment:
Builders should receive only a single instance of the current Rib's `Dependency` defined in the main interface.

```suggestion
    private val dependency: Dependency
```